client/src/client-only-routes/show-settings.tsx

```diff
@@ -195,13 +196,14 @@ export function ShowSettings(props: ShowSettingsProps): JSX.Element {
             isJsAlgoDataStructCert={isJsAlgoDataStructCert}
             isMachineLearningPyCertV7={isMachineLearningPyCertV7}
             isQaCertV7={isQaCertV7}
+            isRelationalDatabasesCertV8={isRelationalDatabasesCertV8}
             isRespWebDesignCert={isRespWebDesignCert}
             isSciCompPyCertV7={isSciCompPyCertV7}
             username={username}
             verifyCert={verifyCert}
           />
-          {showUpcomingChanges && <Spacer />}
-          {showUpcomingChanges && <WebhookToken />}
+          {deploymentEnv !== 'live' && <Spacer />}
+          {deploymentEnv !== 'live' && <WebhookToken />}
```

Comment:
```suggestion
          {deploymentEnv == 'staging' && <Spacer />}
          {deploymentEnv == 'staging' && <WebhookToken />}
```
It's more direct and defaults to the 'safe' behaviour (i.e. the behaviour we want to keep out of production).

Reply:
Thanks, you're probably right that we should default to the safe behavior. Only issue is that we may want to check if it's 'development' as well so we can see it there.

Reply:
Right, but we have staging vs live and development vs production. The dev default being staging + development.

It only gets tricky locally when you try to switch from development to production, since you need secret keys.

Reply:
changes added